content/jobs.md

```diff
@@ -5,40 +5,60 @@ aliases = [ "node/785" ]
 
 The ReactOS Project is seeking qualified individuals to fill multiple positions. Contracted **full time, part time, and scholarship (internship)** level positions are all available.
 
+If you qualify for a position, you'll be joining the very small and exclusive ranks of people who know how to do NT systems development. And believe us, those ranks are very, very small so you will be gaining an immense amount of knowledge working with such people. It's a great way for beginners to interact with people who have been working in this field for over the years.
 
 Available Positions
 ===
 
 Developer
 ---
-Some text about developer qualifications
+We are looking for people proficient in C and its a big plus if you are proficient in C++ too.C++ proficiency does not equate C proficiency so do not assume you have the latter if you have the former. Also keep in mind, again, that these are the minimum requirements. On top of these you need to be a good problem solver, belief in the mission of ReactOS, and most importantly, be able to demonstrate that you can work with the ReactOS codebase via a few patches.
 
+In ReactOS, we need all kinds of people, those who are ready to port drivers and mainly work in the kernel region and those of you who would like to work in user-mode.For all the work related to kernel and drivers, you need to be proficient in C and don't need C++ for most of it. But when it comes to user-mode, you do need to have knowledge of C++. As an example, most of the applications are written in C++ not in C.Whereas when we have a look at the drivers we notice that they are written in C.
```

Comment:
On https://master.web-content.reactos.org/developing/ (temp url) there is a rough explanation about the types of developers / development work

Reply:
Rather than explaining it all here in the positions page wouldn't it be better just to link it up? @learn-more.
We could maybe add a line that says "For detailed information on what we do, refer -[Development Information](https://master.web-content.reactos.org/developing/)". What do you say about this?

Reply:
> In ReactOS, we need all kinds of people, those who are ready to port drivers and mainly work in the kernel region and those of you who would like to work in user-mode. For all the work related to kernel and drivers, you need to be proficient in C and don't need C++ for most of it. But when it comes to user-mode, you do need to have knowledge of C++. As an example, most of the applications are written in C++ not in C.Whereas when we have a look at the drivers we notice that they are written in C.

would be changed to 

> In ReactOS, we need all kinds of people like, those who are ready to port drivers and mainly work in the kernel region, those of you who would like to **work in different other things such as System Processes, Shell and Explorer and Win32 API. If you would like to have a detailed look at what we do, you can refer to our [page](https://master.web-content.reactos.org/developing/) which explains it all in detail.** For all the work related to kernel and drivers, you need to be proficient in C and don't need C++ for most of it. But when it comes to user-mode, you do need to have knowledge of C++. As an example, most of the applications are written in C++ not in C.Whereas when we have a look at the drivers we notice that they are written in C.

Reply:
yeah seems good

Reply:
Cool will finish that part up :)